model/src/main/java/org/projectnessie/api/v2/params/ReferencesParams.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.api.v2.params;
+
+import java.util.Objects;
+import java.util.StringJoiner;
+import javax.annotation.Nullable;
+import javax.ws.rs.QueryParam;
+import org.eclipse.microprofile.openapi.annotations.media.ExampleObject;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.projectnessie.api.params.FetchOption;
```

Comment:
isnt it odd that this class is the only one remaining below `api` ?
will we keep it there for eternity i.e. even if one day we can remove `api.v1` ?

Reply:
It is odd - I commented in the commit about that. It looks like a mistake to me that it was originally added under `param`. It should have been under `model`, IMHO. However, it is reference in java client API, so we cannot move it without breaking binary backward compatibility.

Maybe we can clean it up for 1.0 :thinking: